src/base/projects/SourceBrowser/FileDiff.svelte

```diff
@@ -147,6 +155,7 @@
     </div>
     <div class="browse clickable" on:click|stopPropagation={() => dispatch("browse", file.path)}>
       <Icon name="browse" width={20} inline fill />
+      <span class="secondary view text-xsmall">View file</span>
```

Comment:
This should be an actual tooltip, right now it always shows up. Perhaps just `title="View file"` would work

Reply:
Yeah I went with the solution brandon wrote down
> Some text next to the icon would be ideal. For example "<> View file"

But showing the text as tooltip leaves the design more lean.